### PR TITLE
Added skeleton to cohorts page

### PR DESCRIPTION
**Pull request by korgan00**
https://user-images.githubusercontent.com/4138279/209170550-bf146edc-9226-4d97-8eda-40fd816e5bbc.mov

### CODE REVIEW

**Review by abdonrd — APPROVED** [review]
Looks good!

File: pages/account/teach/course/[course-slug]/edit/index.vue [hunk 1]
```diff
@@ -2,10 +2,36 @@
...
+        </AccountHeaderSection>
+
+        <div class="teach-course-edit__cohort-list">
+          <div
+            class="teach-course-edit__cohort teach-course-edit__loading__cohort"
+          >
+            <bx-skeleton-text />
+          </div>
+          <div
+            v-for="n in 3"
```
Comment (techtolentino) [inline pages/account/teach/course/[course-slug]/edit/index.vue:23]: @korgan00 what are your thoughts on having this as just showing 1 loading skeleton...

on a previous team, we used the rule that a skeleton load should show the _minimum_ amount of information that a user can expect.

the reason: 

for example, it's unrealistic if we show 3 loading boxes, then what eventually renders is 1 box.

in the screenshot shared here, we see 3 Cohorts "loading", but ultimately only see 2...

what do you think?

Comment (abdonrd) [reply pages/account/teach/course/[course-slug]/edit/index.vue:23]: Right! I'm agree!

Comment (abdonrd) [reply pages/account/teach/course/[course-slug]/edit/index.vue:23]: Since Goyo is on vacation, I'm going to take it to be able to merge quickly.

**Review by abdonrd — APPROVED** [review]

**Review by techtolentino — APPROVED** [review]
Thanks @abdonrd!  🚢

**Pull request merged**